src/ontology/mondo-edit.obo

```diff
@@ -518029,14 +518031,11 @@ is_a: MONDO:0003847 {source="OMIM:619318"} ! hereditary disease
 id: MONDO:0036192
 name: obsolete EN1-related dorsoventral syndrome
 subset: ordo_disorder {source="Orphanet:611223"}
-synonym: "ENDOVE syndrome" EXACT [Orphanet:611223]
-synonym: "ENDOVES" EXACT [Orphanet:611223]
-xref: GARD:18027 {source="MONDO:obsoleteEquivalent"}
-xref: Orphanet:611223 {source="MONDO:obsoleteEquivalent"}
 property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeGrouping"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/5114" xsd:anyURI
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6877" xsd:anyURI
```

Comment:
please add the GH #7574  issue here too.

Reply:
Comment addressed